uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list.spec.js

```diff
@@ -321,6 +321,28 @@ define([
       });
     });
 
+    describe('applySidebarFilters()', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      it('does not highlight the select dates filter section', function () {
```

Comment:
Organization could be better
`describe('before applySidebarFilters()' is called` and
`describe('after applySidebarFilters()' is called`